docs/sources/enterprise/auditing.md

```diff
@@ -192,17 +200,42 @@ external group.
 | Disable permissions for datasource                 | `{"action": "disable-permissions", "resources": [{"type": "datasource"}]}`                |
 | Grant datasource permission to role, team, or user | `{"action": "create", "resources": [{"type": "datasource"}, {"type": "dspermission"}]}`\* |
 | Remove datasource permission                       | `{"action": "delete", "resources": [{"type": "datasource"}, {"type": "dspermission"}]}`   |
+| Enable caching for datasource                      | `{"action": "enable-cache", "resources": [{"type": "datasource"}]}`                       |
+| Disable caching for datasource                     | `{"action": "disable-cache", "resources": [{"type": "datasource"}]}`                      |
+| Update datasource caching configuration            | `{"action": "update", "resources": [{"type": "datasource"}]}`                             |
```

Comment:
Not sure if this should be added or just included in the previous update line: https://github.com/grafana/grafana/blob/main/docs/sources/enterprise/auditing.md?plain=1#L189